src/utils/getNormalizedOptions.ts

```diff
@@ -1,23 +1,42 @@
 import { loader } from 'webpack';
 import { getOptions } from 'loader-utils';
 
-import { StyleResourcesInjector, StyleResourcesLoaderOptions, StyleResourcesLoaderOriginalOptions } from '../';
+import { StyleResourcesLoaderOptions, StyleResourcesLoaderOriginalOptions, SpecialInjectors } from '../';
 import { isString } from './';
 
-export function getNormalizedOptions(this: loader.LoaderContext): StyleResourcesLoaderOptions {
-    const defaultInjector: StyleResourcesInjector = (source, resources) =>
-        resources.map(({ content }) => content).join('') + source;
+function fixEndLine(content: string): string {
```

Comment:
IMO, `fixEndLine` is probably useless. It seems more likely some sort of hack here. Shall we remove this and keep it simple, stupid?